<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeLimitDialog</class>
 <widget class="QDialog" name="SizeLimitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Exclude sizes</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="bLayout" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>122</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>206</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="bCancel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bOK">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cExcludeUpperThan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>176</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exclude files upper than</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cExcludeLowerThan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>174</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exclude files lower than</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>153</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>16</number>
    </property>
    <item>
     <widget class="QLineEdit" name="eUpperThan">
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbExcludeUpperUnit">
      <property name="minimumSize">
       <size>
        <width>73</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>154</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>17</number>
    </property>
    <item>
     <widget class="QLineEdit" name="eLowerThan">
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbExcludeLowerUnit">
      <property name="minimumSize">
       <size>
        <width>73</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
